clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/DeliverArraysExecutor.scala

```diff
@@ -78,6 +81,84 @@ class DeliverArraysExecutor(deliverCommand: DeliverArrays)(implicit ec: Executio
         )
     }
   }
+
+  override protected[dispatch] def buildMove(
+    metadata: moveCommand.index.MetadataType,
+    ioUtil: IoUtil
+  ): Source[(moveCommand.index.MetadataType, immutable.Seq[IoOp]), NotUsed] = {
+
+    val baseStream = buildDeliveryMove(metadata, ioUtil)
+      .orElse(Source.single(metadata -> immutable.Seq.empty))
+
+    baseStream.flatMapConcat {
+      case (movedMetadata, moveOps) =>
+        buildDelivery(
+          movedMetadata.withWorkspaceName(deliverCommand.workspaceName),
+          moveOps
+        )
+    }
+  }
+
+  private[dispatch] def buildDeliveryMove(
```

Comment:
This is a wholesale copy from the base class - probably could be done a lot better, but couldn't quickly find a way to only get the pathsForDelivery without filtering them out.

Reply:
Rough idea: add a `deliverInto` method to the [`DeliverableMetadata` trait](https://github.com/broadinstitute/clio/blob/develop/clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/DeliverableMetadata.scala), and call it here.